### DISCUSSION TITLE
If it possible move plots from one account to another (different mnemonic) ?

### DISCUSSION THREAD

**Discussion by dungla2011** [Chia-Network/chia-blockchain#6415] [General]
I have 2 PC, with 2 account (different 24 mnemonic word)
If I can move plots from PC1 to PC2?
If each Plot has a signature which match only with one account?
Thank you!

**Comment by lvcivs** [comment]
No, unless you specifically plotted from the CLI using the farmer and pool public keys from the other computer. Plots are always tied to a specific set of keys.

**Comment by zacmiony** [comment]
One of the users might found something interesting in this topic. I recommend to take a look.
 [https://www.youtube.com/watch?v=mlmtr6VEFOc](https://www.youtube.com/watch?v=mlmtr6VEFOc)

**Reply by ghost** [reply]
Interesting video indeed.. Sadly the scripts are not public. :-)

**Reply by zacmiony** [reply]
I'm afraid that a good thing about this.

**Reply by loppefaaret** [reply]
Don't do this - you're ruining the plot - the moment you get a valid good proof, and try to sign it with a different private key, it is going to fail! you are never going to successfully farm a block doing this!
